src/Diagnostics.DataProviders/DataProviders/KustoDataProvider.cs

```diff
@@ -93,6 +101,51 @@ public async Task<DataTable> ExecuteQuery(string query, string stampName, string
             return await _kustoClient.ExecuteQueryAsync(Helpers.MakeQueryCloudAgnostic(_kustoMap, query), _kustoMap.MapCluster(cluster) ?? cluster, _kustoMap.MapDatabase(_configuration.DBName) ?? _configuration.DBName, requestId, operationName);
         }
 
+        public async Task<DataTable> ExecuteQueryOnAllAppAppServiceClusters(string query, string requestId = null, string operationName = null)
+        {
+            if (string.IsNullOrWhiteSpace(operationName))
+            {
+                throw new ArgumentNullException(nameof(operationName), "OperationName cannot be empty. Please supply a name to identify the query.");
+            }
+            if (string.IsNullOrWhiteSpace(query))
+            {
+                throw new ArgumentNullException(nameof(query), "Query cannot be empty. Please supply a query to execute.");
+            }
+            else
+            {
+                List<Task<DataTable>> queryTask = new List<Task<DataTable>>();
+
+                if(IsPublicCloud)
```

Comment:
Instead of using the cloudenvironment as a configuration to determine if this should be enabled or not can we create an enable flag? Kind of like https://github.com/Azure/Azure-AppServices-Diagnostics/blob/b688b57942342d035a6c9bb3fa7a65c259159168/src/Diagnostics.RuntimeHost/appsettings.json#L60-L61

This will enable us to:
1. Disable in production in the case something weird happens.
2. Enable/Disable anywhere in sovereign clouds when we want.